src/Serilog/Formatting/Display/OutputProperties.cs

```diff
@@ -30,6 +30,11 @@ public static class OutputProperties
     /// </summary>
     public const string TimestampPropertyName = "Timestamp";
 
+    /// <summary>
+    /// The timestamp of the log event in UTC.
+    /// </summary>
+    public const string UTCTimestampPropertyName = "UTCTimestamp";
```

Comment:
can you case the name (and the string) as `Utc` please per FDG naming standards
```suggestion
    public const string UtcTimestampPropertyName = "UtcTimestamp";
```

Reply:
Done